### PR TITLE
[SD-2201] No filter matches all articles

### PR DESCRIPTION
**Pull request by marwoodandrew**
Change the action of not having a filter to pass content.

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: server/apps/content_filters/content_filter_tests.py [hunk 1]
```diff
@@ -312,7 +312,7 @@ def test_build_elastic_query_using_like_filter_multi_content_filter4(self):
 
     def test_does_match_returns_false_for_nonexisting_filter(self):
         for article in self.articles:
-            self.assertFalse(self.f.does_match(None, article))
+            self.assertTrue(self.f.does_match(None, article))
```
Comment (plamut) [inline server/apps/content_filters/content_filter_tests.py]: The name of this test case method should be updated as well to avoid confusion. :)

**Comment by plamut** [comment]
Looks good, just the test case name needs to be changed, too.

:+1:

File: server/apps/content_filters/content_filter_tests.py (hunk 1 above)
Comment (marwoodandrew) [inline server/apps/content_filters/content_filter_tests.py]: Yes, I my mind I'd done that!!! Will fix.

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**